### PR TITLE
PR for Issue Dap-25

### PR DESCRIPTION
**Pull request by jamesleemeredith**

### CODE REVIEW

File: Section09/9.3-Tableau.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+  - **include()** - increases aggregation
+  - **exclude()** - increases granularity
+  - **fixed ()** - increases independence
+
+---
+
+## SUMMARY:
+
+In this section we discussed some of the basic graph types in Tableau, as well as begin a brief discussion on custom calculations.
+
```
Comment (HollymRobinson) [inline Section09/9.3-Tableau.md:203]: On the last day of Tableau we will go more in-depth over the topic of levels of detail. May you can make a statement in this summary about what will be discussed on the fourth day. I will also add to my DAP-23 summary a statement about what the students should expect on day two.

Good job! I was looking for spelling errors or capitalization errors and did not spot any.

**Review by HollymRobinson — COMMENTED** [review]
Good Job!

File: Section09/9.3-Tableau.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+- Three different expressions:
+  - **include()** - increases aggregation
+  - **exclude()** - increases granularity
+  - **fixed ()** - increases independence
+
+---
+
+## SUMMARY:
+
+In this section we discussed some of the basic graph types in Tableau, as well as begin a brief discussion on custom calculations.
```
Comment (HollymRobinson) [inline Section09/9.3-Tableau.md:202]: I copied and pasted these lines. Now I am formatting my day 4

File: Section09/9.3-Tableau.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+
+In this section we'll discuss some of the basic graph types in Tableau, as well as begin a brief discussion on custom calculations.
+
+---
+
+## SCATTER PLOTS
+
+- Used to show the correlation between two measures​ (does not allow dimensions)
+- Create magic quadrants and identify the relationships between the measures
+- Both axes will must be numerical fields​
```
Comment (JohanBester) [inline Section09/9.3-Tableau.md:15]: Did you mean ...
"Both axes must be numerical fields"

File: Section09/9.3-Tableau.md [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+- Change axis range of Profit, right-click on [**Profit**] and select [**Synchronize Axis​**]
+- Change type of Mark for [**Sales**] to bar
+
+<br>
+
+## FUNNEL CHARTS
+
+- Normally used in Marketing & Sales​
+- Helps present sales, profit, and revenue at different stages​
+- Created for single a measure value and multiple measure value​s
```
Comment (JohanBester) [inline Section09/9.3-Tableau.md:103]: Did you mean ...
"Created for a single measure ... "

**Review by JohanBester — DISMISSED** [review]
Great job James!
I love what you have done.
I made 2 comments about word order but these are not issues.

**Review by JohanBester — APPROVED** [review]
Approved

**Pull request merged**